### ISSUE TITLE
feature request: bundle with `importHelpers` (`tslib`)

### ISSUE THREAD

**Issue by JounQin** [markuplint/markuplint#208]
https://www.typescriptlang.org/tsconfig#importHelpers

`markuplint-sync` depends on `markuplint`, not using `importHelpers` may increase the final size of output because of duplicate implementations. And when `markuplint-sync` enables `importHelpers`, they may share different implementations of `__classPrivateFieldGet`, this will make downleveled private fields error.

I'll disable `importHelpers` in `markuplint-sync` temporarily, but I think it should be enabled.

**Comment by YusukeHirao** [comment]
@JounQin 

In the short, It's a problem that `markpulint` package has only downlevel JavaScript code, isn't it? Do you think it becomes to resolve if publishes _MJS_?

**Comment by JounQin** [comment]
No, it depends the target, the current target is es2017, `.mjs` doesn't mean private field is supported, private field or class field is only proposal stage 3 currently.

**Comment by JounQin** [comment]
It's just easy to enable a `importHelpers` flag in `tsconfig.json`.

**Comment by YusukeHirao** [comment]
@JounQin 

Oh yeah, I see. For **private field**, Got it.

Each entry file of all packages need:

```ts
import 'tslib';
```

And `importHelpers` flag in `tsconfig.json` that placed in root.
Last, Add `tslib` to dependencies.

Is this right?

**Comment by JounQin** [comment]
@YusukeHirao Right~

**Comment by YusukeHirao** [comment]
@JounQin 

Oh, I got it. 
I just misremembered it.

Would you do PR?

**Comment by JounQin** [comment]
If you agree to add this, I'd love to PR.

**Comment by YusukeHirao** [comment]
@JounQin 

I agree, thank you!
